src/pages/injector/app.js

```diff
@@ -35,7 +35,7 @@ export const App = () => {
   }
 
   useEffect(async () => {
-    let newTheme = await getSetting('theme')
+    let newTheme = await getSetting('theme') || 'light'
```

Comment:
@wtfluckey This was the culprit that was giving you `undefined`

Reply:
Aha! *shakes fist at sky*